tests/serializers/property/test_property_serializer.py

```diff
@@ -16,8 +16,7 @@ def test_serializer(self, create_property):
             'city': property.city,
             'state': property.state,
             'zipcode': property.zipcode,
-            'dateAdded': property.dateAdded,
             'archived': property.archived,
-            'created_at': Time.format_date(property.created_at),
-            'updated_at': Time.format_date(property.updated_at)
+            'created_at': Time.serialized_date_format(property.created_at),
+            'updated_at': None
```

Comment:
"None" is just for these tests because we are immediately creating the object and then serializing it, which means there won't be an updated_at value since there isn't a default for it.